crypto/src/main/scala/org/bitcoins/crypto/DLEQUtil.scala

```diff
@@ -44,25 +49,17 @@ object DLEQUtil {
   def dleqProve(
```

Comment:
scaladoc: https://github.com/discreetlogcontracts/dlcspecs/blob/d01595b70269d4204b05510d19bba6a4f4fcff23/ECDSA-adaptor.md#proving

Since our parameters aren't the same as the spec, it isn't clear to me where we are enforcing this invariant: 

>x: A non-zero scalar representing the witness for the proof.

It seems if a library is using this independent of the rest of our code base, this is possible?

Reply:
Added the invariant (it was being enforced on all bitcoin-s calls but you're right that that could be a footgun for an outside user